packages/firestore/__tests__/firestore.test.ts

```diff
@@ -658,8 +679,10 @@ describe('Firestore', function () {
 
     it('`getPersistentCacheIndexManager` is properly exposed to end user', function () {
       expect(getPersistentCacheIndexManager).toBeDefined();
-      const indexManager = getPersistentCacheIndexManager(firebase.firestore());
-      expect(indexManager!.constructor.name).toEqual('FirestorePersistentCacheIndexManager');
+      // FIXME there is currently no way to programmatically alter
+      //       persistence settings via modular API (FirestoreSettings.localCache ...)
```

Comment:
This looked like a really large oversight to me. I was unable to do a modular test where I got a cache index manager back, nor was I able to change it's properties etc. Logged as #8309